src/org/omegat/core/dictionaries/LingvoDSL.java

```diff
@@ -29,16 +29,17 @@
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileInputStream;
+import java.io.IOException;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
 import java.util.List;
 import java.util.Locale;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import java.util.zip.GZIPInputStream;
 
+import org.apache.commons.io.input.BOMInputStream;
```

Comment:
This is a nice find. There's a bunch of hand-written BOM detection in OmegaT that we could replace with this.

Reply:
(I'm not suggesting you do that in this PR.)